lantern.go

```diff
@@ -153,12 +157,17 @@ func run(configDir string, user UserConfig) {
 		log.Errorf("Error parsing boolean flag: %v", err)
 	}
 
+	// Need to manually enable logging since we don't have settings in Mobile
+	// TODO: allow configuring whether or not to enable reporting (just like we
+	// already have in desktop)
+	logging.SetReportingEnabled(true)
```

Comment:
Oh, just saw this comment. We technically do have a ``settings.yaml`` on mobile (it's in the ``lantern-mobile/app/src/main/res/raw`` directory), but we haven't really been using it. Easy enough to resurrect it though if we want to configure logging in a similar manner to desktop.